cryosmsSup/cryosms.proto

```diff
@@ -0,0 +1,17 @@
+ReplyTimeout = 200;
```

Comment:
I think these are mismatches between the protocol defined here and what is in the emulator, as I get errors of the following format when I run the IOC tests:

```
2019/11/05 22:41:45.939892 STREAM_PORT TE:NDW1799:CRYOSMS_01:OUTPUTMODE: got "UNITS: AMPS<0d><0a>" where " UNITS: " was expected
2019/11/05 22:41:46.361094 STREAM_PORT TE:NDW1799:CRYOSMS_01:HIDDEN:OUTPUTMODE:SP: 2 bytes surplus input "<0d><0a>"
2019/11/05 22:41:46.361108 STREAM_PORT TE:NDW1799:CRYOSMS_01:HIDDEN:OUTPUTMODE:SP: after 21 bytes "...2:41:46 UNITS: TESLA"
2019/11/05 22:41:46.940802 STREAM_PORT TE:NDW1799:CRYOSMS_01:OUTPUTMODE: Input "........UNITS: TESLA" mismatch after 8 bytes
2019/11/05 22:41:46.940840 STREAM_PORT TE:NDW1799:CRYOSMS_01:OUTPUTMODE: got "UNITS: TESLA<0d><0a>" where " UNITS: " was expected
```

(I am not sure which version is correct; it might be that you need to change the emulator rather than this file)